File: snapshots/input/syntax/Main/Structs.cs
using System.Diagnostics.CodeAnalysis;

namespace Main;

[SuppressMessage("ReSharper", "all")]
public class Structs
{
    struct BasicStruct
    {
        public int Property1;

        public BasicStruct(int field1)
        {
            Property1 = field1;
        }
    }
}

File: snapshots/input/syntax/VBMain/Structs.vb
Imports System.Diagnostics.CodeAnalysis

Namespace VBMain
    <SuppressMessage("ReSharper", "all")>
    Public Class Structs
        Structure BasicStruct
            Public Property1 As Integer

            Public Sub New(ByVal field1 As Integer)
                Property1 = field1
            End Sub
        End Structure
    End Class
End Namespace
